dev_modules/custom/health_sample_content/config/install/migrate_plus.migration.sample_node_standard_page_form.yml

```diff
@@ -0,0 +1,156 @@
+id: sample_node_standard_page_form
```

Comment:
@Tjocksmurf - given we don't have a Standard Page Form content type I would either add the webform component to an existing sample Stardard Page content page or create a new sample Standard Page content page in the existing Standard Page migration template. Will leave you to decide which of these two options makes the most sense.